### PR TITLE
docs: fix broken links

### PR DESCRIPTION
**Pull request by sommeeeer**

### CODE REVIEW

File: docs/pages/inner_workings.mdx [hunk 1]
```diff
@@ -2,7 +2,7 @@ When calling `open-next build`, OpenNext **runs `next build`** to build the Next
 
 #### Building the Next.js app
 
-OpenNext runs the `build` script in your `package.json` file. Depending on the lock file found in the app, the corresponding packager manager will be used. Either `npm run build`, `yarn build`, or `pnpm build` will be run. For more on customizing the build command, see [overriding the build command](/advanced/options#custom-build-command).
+OpenNext runs the `build` script in your `package.json` file. Depending on the lock file found in the app, the corresponding packager manager will be used. Either `npm run build`, `yarn build`, or `pnpm build` will be run. For more on customizing the build command, see [overriding the build command](/config#build-arguments).
```
Comment (conico974) [inline docs/pages/inner_workings.mdx:5]: This is not the correct link, to override the build command you have to use the `buildCommand` fields in the `open-next.config.ts` https://open-next.js.org/config/full_example

**Review by khuezy — APPROVED** [review]
Thank  you!

File: docs/pages/inner_workings.mdx (hunk 1 above)
Comment (sommeeeer) [reply docs/pages/inner_workings.mdx]: > This is not the correct link, to override the build command you have to use the `buildCommand` fields in the `open-next.config.ts` https://open-next.js.org/config/full_example

image
 This better?

Comment (conico974) [reply docs/pages/inner_workings.mdx]: Yeah perfect

**Review by conico974 — APPROVED** [review]
Thanks

**Pull request merged**